### DISCUSSION TITLE
Using tiddlyclip to embed a webpage with iframe

### DISCUSSION THREAD

**Discussion by arunnbabu81** [buggyj/tiddlyclip-plugin#67] [Q&A] [answered]
I would like to embed webpages in tiddlywiki using iframe with the help of tiddlyclip.
How should i frame the snip rule to get this code into the tiddler body

`<html><div align="center"><iframe src=""frameborder="0" width="100%" height="800"></iframe></div></html>`

**Comment by arunnbabu81** [comment]
Is there anything that can be done for this

**Comment by buggyj** [comment, marked as answer]
you can copy the 'snip'. In the 'body' column we have

`((*@pageRef*))\n\n((*@text*))\n\n`

substitute with 

`<div align="center"><iframe src='((*@pageRef*))' frameborder='0' width='100%' height='800'></iframe></div>`

**Reply by arunnbabu81** [reply]
> you can copy the 'snip'. In the 'body' column we have
> 
> `((*@pageRef*))\n\n((*@text*))\n\n`
> 
> substitute with
> 
> `<div align="center"><iframe src='((*@pageRef*))' frameborder='0' width='100%' height='800'></iframe></div>`

![image](https://user-images.githubusercontent.com/67494083/152418495-4a6eaf68-603e-4db0-82d6-0426dd9348b1.png)

![image](https://user-images.githubusercontent.com/67494083/152418092-2056684f-de03-4c3f-8972-7ed043039a09.png)


This doesn't seem to work

**Reply by buggyj** [reply]
try
`<div align='center'><iframe src='((*@pageRef*))' frameborder='0' width='100%' height='800'></iframe></div>`

**Reply by arunnbabu81** [reply]
> try `<div align='center'><iframe src='((*@pageRef*))' frameborder='0' width='100%' height='800'></iframe></div>`

This one worked. thank you once again

**Reply by arunnbabu81** [reply]
I was combing tiddlyclip and dynamic tables from shiraz plug in to make a bookmark list or reading list. This was for that purpose
